integration_tests/test_helpers/helpers.go

```diff
@@ -150,15 +150,12 @@ func ReadAndTestPublisherData(dataFilePath string, expectedMetrics map[string]st
 // soMetricRowIsValid function takes 3 strings:
 // namespace, tags, value - and checks if provided value is almost equal
 // corresponding one from expectedMetrics
-func soMetricRowIsValid(expectedMetrics map[string]string, namespace, tags, value string, session *snap.Session) {
+func soMetricRowIsValid(expectedMetrics map[string]string, namespace, tags, value string) {
 
 	// Check tags.
 	tagsSplitted := strings.Split(tags, ",")
 	convey.So(len(tagsSplitted), convey.ShouldBeGreaterThanOrEqualTo, 1)
 
-	convey.So(session.GetTags(), convey.ShouldContainKey, "foo")
```

Comment:
Because this test does not assert that any tag has been saved or sent to Snap.
Simply, this test does not test anything.